src/Console/CodegenCommand.php

```diff
@@ -5,24 +5,39 @@
 use Spawnia\Sailor\Codegen\Generator;
 use Spawnia\Sailor\Codegen\Writer;
 use Spawnia\Sailor\Configuration;
+use SplFileInfo;
 use Symfony\Component\Console\Command\Command;
 use Symfony\Component\Console\Input\InputArgument;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
 
 class CodegenCommand extends Command
 {
+    private const OPTION_CONFIGURATION = 'configuration';
+
     protected static $defaultName = 'codegen';
 
     protected function configure(): void
     {
         $this->setDescription('Generate code from your GraphQL files.');
         $this->addArgument('endpoint', InputArgument::OPTIONAL, 'You may choose a specific endpoint. Uses all by default.');
+        $this->addOption(
+            self::OPTION_CONFIGURATION,
+            'c',
+            InputArgument::OPTIONAL,
+            'File to read configuration from. Default `sailor.php`.',
+            'sailor.php'
+        );
     }
 
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
         $endpoint = $input->getArgument('endpoint');
+        $configurationFile = $input->getOption(self::OPTION_CONFIGURATION);
+        assert(is_string($configurationFile));
```

Comment:
That might not necessarily hold true, I think a client could pass a comma-separated list or just a number.

Reply:
> a comma-separated list or just a number.

that will still be interpreted as a string, no?

Reply:
Only one way to find out 🤷 I know that in Laravel, console args/options can be non-strings.

Reply:
It has to be interpreted as a string, otherwise it would make no sense. `--arg 1` must be string since cli cannot read minds. If app wants a number it must validate the input and eventually convert it.